src/sparsezoo/model.py

```diff
@@ -316,6 +306,26 @@ def __str__(self):
         else:
             return f"{self.__class__.__name__}(directory={self.source})"
 
+    def initialize_model_from_stub(
+        self, stub: str
+    ) -> Tuple[List[Dict[str, str]], str, str]:
+        files, model_id, params = load_files_from_stub(
+            stub, valid_params=list(PARAM_DICT.keys())
+        )
+        if params:
+            self._validate_params(params)
+        path = os.path.join(CACHE_DIR, model_id)
```

Comment:
let's talk about ways users can override the cache dir (ie in the previous API there was an environment variable for the default sparsezoo download location + a path override in the download function signature). we should also have a force re-download option

Reply:
It's quite simple to be honest.
It's not being done during the construction of the object, since we agreed to have a "monadic" constructor for the `Model`. To override cache directory, we simply go

```python
from sparsezoo import Model

stub = ""
directory_to_overwrite = ""

model = Model(stub)
# alternative one, no downloading
model.path = directory_to_overwrite 
# alternative two, downloading
model.download(directory_path=directory_to_overwrite)
```

does it make sense?

Reply:
force re-download is enforced by:
```python
model.download(..., override= True)
```
well, I should have named it `overwrite` rather then `override` I guess...